### DISCUSSION TITLE
Not all NFT explorers support Polygon

### DISCUSSION THREAD

**Discussion by chrishobcroft** [NFTsTv/Broadcastr#56] [Ideas]
My objective is to somehow "certify" NFT explorer dApps, as being "livestream-NFT-ready".

But some platforms don't support Polygon.

One way to be able to test on "all" platforms would be to deploy an NFT to mainnet, and then test which sites are "livestream-NFT-ready".

**Comment by pblvrt** [comment]
The latest tagline for Broadcastr mentions "infinite platforms". At some point we might therefore wish to enable users to mint elsewhere than only Polygon.

    Ethereum Mainnet
    Optimism
    Arbitrum
    Gnosis Chain
    zksync
    Binance

This issue is to open a conversation about what it would take, and what dependencies there might be for doing this.
